src/screens/StarterPack/StarterPackScreen.tsx

```diff
@@ -73,26 +79,55 @@ export function StarterPackScreen({route}: StarterPackScreeProps) {
     isLoading: isLoadingStarterPack,
     isError: isErrorStarterPack,
   } = useStarterPackQuery({did, rkey})
+  const listMembersQuery = useListMembersQuery(starterPack?.list?.uri, 50)
 
   const isValid =
     starterPack &&
     AppBskyGraphDefs.validateStarterPackView(starterPack) &&
     AppBskyGraphStarterpack.validateRecord(starterPack.record)
 
-  if (!did || !starterPack || !isValid || !moderationOpts) {
+  if (
+    !did ||
+    !starterPack ||
+    (!listMembersQuery.data &&
+      starterPack?.creator?.did !== currentAccount?.did) ||
+    !isValid ||
+    !moderationOpts
+  ) {
     return (
       <ListMaybePlaceholder
-        isLoading={isLoadingDid || isLoadingStarterPack || !moderationOpts}
-        isError={isErrorDid || isErrorStarterPack || !isValid}
+        isLoading={
+          isLoadingDid ||
+          isLoadingStarterPack ||
+          listMembersQuery.isLoading ||
+          !moderationOpts
+        }
+        isError={
+          isErrorDid ||
+          isErrorStarterPack ||
+          // We still want this to be visible to the creator even if the list is not found. i.e. takedowns
+          (listMembersQuery.data &&
+            starterPack?.creator?.did !== currentAccount?.did) ||
+          !isValid
+        }
         errorMessage={_(msg`That starter pack could not be found.`)}
+        emptyMessage={_(msg`That starter pack could not be found.`)}
       />
     )
   }
 
+  if (
+    !listMembersQuery.data &&
+    starterPack.creator.did === currentAccount?.did
+  ) {
+    return <InvalidStarterPack rkey={rkey} />
+  }
```

Comment:
Will this ever flash as the list loads? Basically, should we use the error state instead of data existence to determine if we should show this

Reply:
i actually tried that the first time, but turns out “list” isn’t even set on the starterPackView after it gets deleted. 

that said though let me switch this to “!starterPack?.list?.uri” so it’s more clear.

re: a flash, don’t see one on my end, though switching it to the above should be a better guarantee of that.